### PR TITLE
Composer: Fix PHP-CSS-Parser patching

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary
Fix css parer composer package from not loading patch fies. 

## Relevant Technical Choices

Loading the composer package in our composer file, so we have more control. 
## Testing Instructions

---

Fixes #

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
     "cweagans/composer-patches": "^1.6",
     "enshrined/svg-sanitize": "^0.14.0",
     "mcaskill/composer-exclude-files": "^2.0",
-    "symfony/polyfill-mbstring": "^1.18"
+    "symfony/polyfill-mbstring": "^1.18",
+    "sabberworm/php-css-parser": "dev-master#bfdd976"
```
Comment (spacedmonkey) [inline composer.json]: Copied from https://github.com/ampproject/amp-wp/blob/9f14d6147428e58d6d770c41fe5e63e8856ef5e9/composer.json#L19

File: composer.json [hunk 2]
```diff
@@ -67,17 +68,9 @@
...
-          "Validate name-start code points for identifier <https://github.com/sabberworm/PHP-CSS-Parser/pull/185>": "patches/php-css-parser-pull-185.patch",
-          "Fix parsing CSS selectors which contain commas <https://github.com/westonruter/PHP-CSS-Parser/pull/1>": "patches/php-css-parser-commit-10a2501.patch"
-        }
+        "1. Add additional validation for size unit <https://github.com/sabberworm/PHP-CSS-Parser/pull/193>": "https://github.com/sabberworm/PHP-CSS-Parser/compare/3bc5ded67d77a52b81608cfc97f23b1bb0678e2f%5E...468da3441945e9c1bf402a3340b1d8326723f7d9.patch",
+        "2. Validate name-start code points for identifier <https://github.com/sabberworm/PHP-CSS-Parser/pull/185>": "https://github.com/sabberworm/PHP-CSS-Parser/compare/d42b64793f2edaffeb663c63e9de79069cdc0831%5E...113df5d55e94e21c6402021dfa959924941d4c29.patch",
+        "3. Fix parsing CSS selectors which contain commas <https://github.com/westonruter/PHP-CSS-Parser/pull/1>": "https://github.com/sabberworm/PHP-CSS-Parser/compare/master...10a2501c119abafced3e4014aa3c0a3453a86f67.patch"
```
Comment (spacedmonkey) [inline composer.json]: Copied from https://github.com/ampproject/amp-wp/blob/9f14d6147428e58d6d770c41fe5e63e8856ef5e9/composer.json#L60-L62

**Review by swissspidy — COMMENTED** [review]
Need to run `composer composer-normalize` but otherwise looks OK.

**Comment by swissspidy** [comment]
> Fix css parer composer package from not loading patch files.

Just to clarify, I think the patches were loaded, but one was not applying cleanly anymore

**Comment by spacedmonkey** [comment]
Btw, I think the reason why the patches started failing, as our version was different from amp plugin version. 
There were two changes. 
https://github.com/ampproject/amp-wp/commit/7a2b66b31f67593613cffb22d2fc85c63d3b6c16
https://github.com/ampproject/amp-wp/commit/fc3514a47a84650818371e72ad02978e42a11c23

Amp plugin has not used local patches in nearly 3 months.

**Review by swissspidy — APPROVED** [review]

**Comment by spacedmonkey** [comment]
This PR seems to update the amp plugin from 2.0.8->2.0.10. Should I revert this change?

File: composer.json [hunk 3]
```diff
@@ -16,6 +16,7 @@
     "cweagans/composer-patches": "^1.6",
     "enshrined/svg-sanitize": "^0.14.0",
     "mcaskill/composer-exclude-files": "^2.0",
+    "sabberworm/php-css-parser": "dev-master#bfdd976",
```
Comment (swissspidy) [inline composer.json:19]: Is this really needed?

File: web-stories.php [hunk 4]
```diff
@@ -42,7 +42,7 @@
 
 define( 'WEBSTORIES_VERSION', '1.3.0' );
 define( 'WEBSTORIES_DB_VERSION', '3.0.6' );
-define( 'WEBSTORIES_AMP_VERSION', '2.0.8' ); // Version of the AMP library included in the plugin.
+define( 'WEBSTORIES_AMP_VERSION', '2.0.10' ); // Version of the AMP library included in the plugin.
```
Comment (swissspidy) [inline web-stories.php:45]: `composer.json` says `2.0.8`. Should we perhaps change the requirement there from `^2.0.8` to a fixed `2.0.10`?

File: composer.json (hunk 3 above)
Comment (swissspidy) [reply composer.json:19]: looks like it is

**Comment by swissspidy** [comment]
> Btw, I think the reason why the patches started failing, as our version was different from amp plugin version.
> There were two changes.
> [ampproject/amp-wp@7a2b66b](https://github.com/ampproject/amp-wp/commit/7a2b66b31f67593613cffb22d2fc85c63d3b6c16)
> [ampproject/amp-wp@fc3514a](https://github.com/ampproject/amp-wp/commit/fc3514a47a84650818371e72ad02978e42a11c23)
> 

Totally missed those upstream changes (which I've requested a a while ago)

This allows us to totally simplify the patching

**Review by swissspidy — APPROVED** [review]

**Comment by spacedmonkey** [comment]
This looks good to go now 👍

**Pull request merged**